ChangeLog.md

```diff
@@ -6,6 +6,7 @@ Note: This file only contains high level features or important fixes.
 
 ### 3.6.0 - Daily Build
 
+* ADSB: Added support for connecting to SBS server. Adds support for ADSB data from USB SDR Dongle running 'dump190 --net' for example.
```

Comment:
should this read: ".... running 'dump1090 --net' for example." ?

Reply:
fixed